### DISCUSSION TITLE
Checklist

### DISCUSSION THREAD

**Discussion by bubugian** [XCSoar/XCSoar#1373] [Ideas]
Could be useful to have a checklist with "check_done_and_OK-buttons"

**Comment by lordfolken** [comment]
I use the checklist extensivly and would love enhancments. 
The checklist is multipage capable
so you can add 
```
[checklist1]
item
item
ietm


[checklist2]
item 
item
```
and then you can switch between the pages.

The checkbox items could be implemented with markdown syntax:
- [ ] item1
- [ ] item2
- [ ] item3

Then going up and down with keyboard/stick should select each indiuvidual item. Hitting enter/ok should mark resp. unmark the item.
